### DISCUSSION TITLE
!leetgo pick ID issue

### DISCUSSION THREAD

**Discussion by zxcvbnjs** [j178/leetgo#285] [Q&A] [answered]
I encounter an issue of using !leetgo pick ID in the jupyter environment of my Mac Laptop and it always says that it can not find this question due to an invalid qid. The !leetgo pick title works but !leetgo pick ID does not work which is very strange. I use the latest version of leetgo but this issue can not be fixed. So I want to ask what might be the issue of my environment?

<img width="1440" alt="1 1" src="https://github.com/j178/leetgo/assets/102926905/3c043d20-e692-4221-a2a9-d935c553a9ac">

**Comment by j178** [comment, marked as answer]
Cound you refresh the local cache by running `leetgo cache update` and see if it works?

**Reply by zxcvbnjs** [reply]
Yes, it works now! Thank you very much!

**Comment by zarmars** [comment]
I also encountered the same problem. But after I run "leetgo cache update" the problem still exists. So I checked the cache directory and found "leetcode-questions.json" was empty. Can you provide some advice? Thank you in advance! @j178

**Reply by j178** [reply]
`leetgo cache update` seems not running successfully, could you try `leetgo cache update` more times?
